<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScenarioTacticalObjects</class>
 <widget class="QWidget" name="ScenarioTacticalObjects">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>496</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ScenarioTacticalObjects</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gbTacticalGroup">
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color:white;
} 
QPushButton:checked{
	background-color: rgb(80, 80, 80);
	border-color: white;
}
QPushButton:hover{
	background-color: grey;
	border-style: outset;
}</string>
     </property>
     <property name="title">
      <string>Тактические знаки</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <layout class="QGridLayout" name="glButton">
        <property name="spacing">
         <number>1</number>
        </property>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnBEK">
          <property name="text">
           <string>БЭК</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="btnBPLA">
          <property name="text">
           <string>БПЛА</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="lwTacticalSigns"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddTacticalSign">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Нанести</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
